### PR TITLE
Delete user profile 2

### PR DESCRIPTION
**Pull request by aeydie**
I created a new branch off of main to hope for less merge requests.  Button will pop up once a user profile has been created as an option on the user profile to delete user.  If button is clicked, user profile is deleted and the user is directed back to the events page.

### CODE REVIEW

File: src/app/user-profile/user-profile.component.html [hunk 1]
```diff
@@ -15,4 +15,9 @@ <h2>{{ currentUser.email }}</h2>
 
     <!-- would be cool to have list of their created events here -->
   <p><button href="#">Log Out</button></p>
+  <p><button routerLink="/events" routerLinkActive="active"
+    class="badge badge-danger mr-2" (click)="deleteUser(currentUser.id)">
+    Delete User?
```
Comment (lfallert) [inline src/app/user-profile/user-profile.component.html]: Can we change this to "Delete Account?"?

File: src/app/user-profile/user-profile.component.ts [hunk 2]
```diff
@@ -33,4 +33,22 @@ export class UserProfileComponent implements OnInit {
     .subscribe(data => this.currentUser = data);
   }
 
+  deleteUser(id: number) {
+    if(confirm(`Are you sure you want to delete this user?  This cannot be undone.`)) {
```
Comment (lfallert) [inline src/app/user-profile/user-profile.component.ts]: Can we get the pop up box to ask "Are you sure you want to delete your account? This cannot be undone."

**Review by lfallert — APPROVED** [review]
works as intended

**Review by CNichole726 — APPROVED** [review]
Works great!

**Pull request merged**